addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -561,4 +570,14 @@
 <template id="opt_event_description_cover_hidden" name="Event Description Cover Hidden Option" inherit_id="website_event.event_description_full" active="False"/>
 <template id="opt_event_description_cover_top" name="Event Description Cover Top Option" inherit_id="website_event.event_description_full" active="False"/>
 
+<!-- Sidebar Blocks Visibility Options -->
+<template id="opt_event_registration_block" name="Registrations" inherit_id="website_event.event_description_full" active="True"/>
+<template id="opt_event_dates_block" name="Dates" inherit_id="website_event.event_description_dates" active="True"/>
+<template id="opt_event_calendar_block" name="Calendar links" inherit_id="website_event.event_description_dates" active="True"/>
```

Comment:
I'm not sure, event_description_full vs event_description_dates in this case ? :p

Reply:
You're right :slightly_smiling_face: Could have been `event_description_full`.
Testing leftovers :smile: